pkg/volume/azuredd/azure_mounter.go

```diff
@@ -102,7 +102,6 @@ func (m *azureDiskMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs) e
 			klog.Errorf("azureDisk - Unmount directory %s failed with %v", dir, err)
 			return err
 		}
-		mountPoint = true
```

Comment:
looks legit!

Reply:
would like azure/storage eyes on this change. cc @msau42 @andyzhangx

Reply:
It doesn't look like this variable is used after this point until https://github.com/kubernetes/kubernetes/pull/99288/files#diff-3dd3d4ae3d9ea57ba5483cdc5e75a8a11ee447dfc748750902ec7c66e0e8f391R138 where it gets reassigned.

So this looks safe to remove.

Reply:
Ack